openreview/conference/builder.py

```diff
@@ -92,12 +92,13 @@ def __init__(self, client):
         self.area_chair_identity_readers = []
         self.senior_area_chair_identity_readers = []
 
-    def __create_group(self, group_id, group_owner_id, members = [], is_signatory = True, additional_readers = []):
+    def __create_group(self, group_id, group_owner_id, members = [], is_signatory = True, additional_readers = [], exclude_self=False):
```

Comment:
Can you call it `exclude_self_reader`? so it is clear we are not making the group.id reader of it self?